lib/galaxy/webapps/galaxy/api/quotas.py

```diff
@@ -145,3 +144,11 @@ def undelete( self, trans, id, **kwd ):
             return self._undelete_quota( quota )
         except ActionInputError as e:
             raise HTTPBadRequest( detail=str( e ) )
+
+
+def get_quota( trans, id ):
```

Comment:
I'm probably missing something but I'm a bit confused by this - where does this get called? All the calls to get_quota seem to be on objects not from modules like this.